File: multilevel_inheritence.py
class A:
    def __init__(self) -> None:
        print("from class A")
    
class B(A):
    def __init__(self) -> None:
        super().__init__()
        print("from class B")

class C(B):
    def __init__(self) -> None:
        super().__init__()
        print("from class C")
    

obj = C()

File: multilevel_inheritence.cpp
#include <bits/stdc++.h>
using namespace std;
class A{
    public:
    void show(){
        cout<<"from class a"<<endl;
    }
};
class B:public A{
    public:
    void show(){
        cout<<"from class b"<<endl;
    }
};
class C:public B{
    public:
    void show(){
        cout<<"from class c"<<endl;
    }
};
int main()
{
    ios_base::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);

    C obj;
    obj.show();
// output the class method 
    return 0;
}